config/IPIQA.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<main>
+<CONFIG>
+    <!-- IPIQA configurations: -->
+    <parameter key="number_of_cores" value= "6"/>
+    <parameter key="experiment_finished" value="True"/>
```

Comment:
number_of_cores and number_of_cores don't describe parameters of computer, but rather execution. Please, remove them from IPIQA.xml